build.gradle

```diff
@@ -144,6 +144,10 @@ dependencies {
 
     compile "com.googlecode.owasp-java-html-sanitizer:owasp-java-html-sanitizer:20180219.1"
 
+    implementation 'org.apache.tika:tika-core:2.4.1'
+    implementation 'org.apache.tika:tika-langdetect:2.4.1'
+    implementation 'org.apache.tika:tika-langdetect-optimaize:2.4.1'
+
```

Comment:
Is it possible to reduce the number of imports via dependencies?

Reply:
Removed import for  `'org.apache.tika:tika-langdetect:2.4.1'` as it is accessible via tika-core.